src/Microsoft.OpenApi.Readers/SchemaTypeConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System;
+using Json.Schema;
+
+namespace Microsoft.OpenApi.Readers
+{
+    internal static class SchemaTypeConverter
+    {
+        internal static SchemaValueType ConvertToSchemaValueType(string value)
+        {
+            return value switch
```

Comment:
are we sure the value is always going to be lowercased or should we normalize it?

Reply:
We can normalize it

Reply:
thanks for making the change, you don't need to affect the variable, you can do the normalization straight in the switch